src/GoogleTranslate.php

```diff
@@ -149,4 +159,17 @@ public function sanitizeLanguageCode(string $languageCode)
             get the list of valid and supported language codes by running GoogleTranslate::languages()"
         );
     }
+
+    protected function validateInput($input): void
+    {
+        if(is_array($input) && in_array(null, $input)) {
+            throw new \InvalidArgumentException('Input string cannot be null');
```

Comment:
can you import it at the top and remove `\` on both lines

Reply:
Pushed amendments